tbx/project_styleguide/templates/patterns/atoms/tag/tag.html

```diff
@@ -1,6 +1,6 @@
-<a  class="tag"
-    href="{{ tag_link_base }}?filter={{ tag_slug }}"
+<a class="tag"
+    href="{{ tag_link_base }}?filter={% firstof item.slug tag_slug %}"
     {% if is_active %}aria-current="page" {% endif %}
 >
-    {% if has_all_filter %}All{% else %}{{ tag_name }}{% endif %}
+    {% if has_all_filter %}All{% else %}{% firstof item.name tag_name %}{% endif %}
```

Comment:
I changed this just on the principle of trying to keep var names in included templates 'flat' as then it is easier to re-use them - is it possible to change how tag.html is included in order to pass through `tag_slug` and `tag_name` separately?

Reply:
Thanks, I must have been in autopilot! 

Updated in https://github.com/torchbox/torchbox.com/pull/41/commits/223de523acc88a4f83d8eaa5e1ca90ad8b6ec50f (also removed `small=True` which wasn't doing anything)